packages/main/src/pages/reset-password/ChangePassword.jsx

```diff
@@ -75,9 +75,19 @@ const NewPassword = () => {
       updatepass();
     }
   };
+  const textAlign = () => {
```

Comment:
don't duplicate this function
it's the same as the one in the other file

Reply:
i already removed it

Reply:
please review it again i already fixed the code